### PR TITLE
fix(file-picker): improve UI for readOnly

### PR DESCRIPTION
**Pull request by LucyChyzhova**
#
Screenshot 2024-08-16 at 10 24 43

# Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/chip-set/partial-styles/_file-picker.scss [hunk 1]
```diff
@@ -79,3 +79,8 @@ $max-width-of-label-when-not-readonly: 40%;
         }
     }
 }
+
+.mdc-text-field.mdc-text-field--disabled.lime-text-field--readonly
+    .lime-looks-like-input-value {
+    padding-left: 1rem;
```
Comment (Kiarokh) [inline src/components/chip-set/partial-styles/_file-picker.scss:85]: this wouldn't work if there is no leading icon.
in file-picker there is a hardcoded leading icon. But chip-set can be with or without the leading icon.

Comment (LucyChyzhova) [reply src/components/chip-set/partial-styles/_file-picker.scss:85]: ahhh! right! thanks! will check the icon

**Review by Kiarokh — APPROVED** [review]

**Pull request merged**